Doc/whatsnew/3.10.rst

```diff
@@ -79,6 +79,9 @@ New Features
   :class:`types.MappingProxyType` object wrapping the original
   dictionary. (Contributed by Dennis Sweeney in :issue:`40890`.)
 
+* PEP 618: The :func:`zip` function now has an optional ``strict`` flag, used
+  to enforce that the all the iterables have an equal length.
```

Comment:
Changed. I followed the guidelines at the top of this file: "Do not spend very much time on the wording of your changes, because your text will probably get rewritten to some degree."